### ISSUE TITLE
eClipse

### ISSUE THREAD

**Issue by Raakh** [jeevansai502/classRoom#1]
I am not able to run this project in eClipse. Can you please let me know how it could be run in eClipse?

**Comment by jeevansai502** [comment]
Hii,
Please come to chat so that i could explain you.

**Comment by Raakh** [comment]
Hello,
Thanks for your prompt reply. How can I chat with you?
Best regards

**Comment by jeevansai502** [comment]
My email jeevansai502@gmail.com.

**Comment by jeevansai502** [comment]
Added readme file with description.

**Comment by Raakh** [comment]
Thanks for adding jars and reame file.
Have you please uploaded its database information anywhere in the project

**Comment by jeevansai502** [comment]
No, you can understand what i am using in the code itself and make database appropriately.
